src/scripts.js

```diff
@@ -210,6 +213,9 @@ function searchFavoriteRecipeByName() {
     filteredFavorites.forEach((current) => {
         displayRecipePreview(current, favoritesView)
     })
+    if(filteredFavorites.length === 0 || searchBar.value === '') {
+       favoritesView.innerHTML = `<p>No recipes found. Please search by recipe name, or select a category to filter recipes.</p>`
```

Comment:
This error message also looks great!

Reply:
It looks like this code is repetitive though, could we make it a helper function and then have a parameter to pass in the view?